include/libc64/math64.h

```diff
@@ -0,0 +1,20 @@
+#ifndef LIBC64_MATH64_H
+#define LIBC64_MATH64_H
+
+#include "ultra64.h"
+
+f32 Math_FTanF(f32 x);
+f32 Math_FFloorF(f32 x);
+f32 Math_FCeilF(f32 x);
```

Comment:
Similar to `qrand.c`, do we have og names for functions from `math64.c`?

Reply:
Sadly no, we know the original names of like 2 functions